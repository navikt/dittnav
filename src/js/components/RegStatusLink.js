import React, { Component } from 'react';
import PropTypes from 'prop-types';

class RegStatusLink extends Component {
  render() {
    return (
      <section className="ditt-list">
      </section>
    );
  }
}

RegStatusLink.propTypes = {
  regStatusLink: PropTypes.string.isRequired,
  erUnderRegistreringIArbeid: PropTypes.bool.isRequired,
};

export default RegStatusLink;
