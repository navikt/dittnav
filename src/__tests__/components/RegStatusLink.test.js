import * as React from 'react';
const ReactTestRenderer = require('react-test-renderer');
import RegStatusLink from '../../js/components/RegStatusLink';

test('RegStatusLink basic test', () => {
  const component = ReactTestRenderer.create((<RegStatusLink />));
  expect(component.toJSON()).toMatchSnapshot();
});
