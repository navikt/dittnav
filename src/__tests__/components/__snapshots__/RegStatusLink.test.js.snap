// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegStatusLink basic test 1`] = `null`;
